libensemble/tests/scaling_tests/balsam_forces/readme.md

```diff
@@ -0,0 +1,61 @@
+## Running test run_libe_forces_balsam.py
+
+Naive Electrostatics Code Test
+
+This is a synthetic, highly configurable simulation function. Its primary use
+is to test libEnsemble's capability to submit application instances via the Balsam service,
+including to separate machines from libEnsemble's processes.
+
+### Forces Mini-App
+
+A system of charged particles is initialized and simulated over a number of time-steps.
+
+Particles' position and charge are initiated using a random stream.
+Particles are replicated on all ranks.
+**Each rank** computes forces for a subset of particles (`O(N^2)` operations).
+Particle force arrays are `allreduced` across ranks.
+Particles are moved (replicated on each rank).
+Total energy is appended to the forces.stat file.
+
+To run forces as a standalone executable on `N` procs:
+
+    mpirun -np N ./forces.x <NUM_PARTICLES> <NUM_TIMESTEPS> <SEED>
+
+This application will need to be compiled on the remote machine where the sim_f will run.
+See below.
+
+### Running with libEnsemble.
```

Comment:
We want to say up front, that you currently need ALCF login credentials to use this service, regardless of what remote machine you are using. 

Then we could say here is an example running libEnsemble on your local computer/laptop and having `forces.x` runs on the remote machine. Making clear which instructions are for laptop and which for remote machine. E.g. Any changes to calling script/sim function are on laptop. But forces.x must be compiled on remote machine.

Reply:
You may want to tell the user to set up their own conda (or other virtual) environment. I first tried the pip install of balsam without that and got a warning could not access /soft/datascience/conda/miniconda3/latest/lib/python3.7/site-packages/.

Also they will need balsam installed on their local machine.